client_app/src/components/admin/AddShelterForm.js

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from 'react';
+import '../../styles/admin/AddShelterForm.css';
+import { SERVER } from "../../config";
+
+const AddShelterForm = () => {
+  const [formData, setFormData] = useState({
+    name: '',
+    street_address_1: '',
+    street_address_2: '',
```

Comment:
```suggestion
    street2: '',
```

Reply:
The server expects this format:
{"name": "some name", "address": {"street1": "1 Grand Ave", "street2": "", "city":"St. Louis", "state": "MO", "postalCode": "12345", "coordinates": {"latitude": 90, "longitude":-30}}}